test/index.ts

```diff
@@ -0,0 +1,73 @@
+
+import { should } from 'chai';
+import { Crypto } from '../src/';
+import Keyring from "@polkadot/keyring";
+
+should();
+
+describe('crypto', () => {
+    it('Test crypto structure', async () => {
+      // Testing result format.
+      const nodes = 5;
+      const c = new Crypto(nodes);
+      const result = await c.createKeys();
+      Object.keys(result).length.should.eq(7);
```

Comment:
Where does this 7 come from? Maybe better export `KeyTypes` in index.ts (extract it from the class) and check its length here.